### PR TITLE
Remove duplicate text from QR code generator page (EN)

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR removes duplicate text from "introCLI".

### CODE REVIEW

File: src/data/eventregistration.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
             "title": "Create QR code for event check-in"
         },
         "intro": "The app now allows a check-in for events and locations. People who host events or have a business can now create a QR code. By scanning the QR code, guests can check in upon arrival to register their presence. If desired, the app will also create a corresponding diary entry. If a checked-in person later tests positive for the coronavirus, other people who were checked in at the same time can be warned.",
-        "introCLI": " You can create QR codes in the app or here on this page. Alternatively, you can also use this command line tool <a href='https://github.com/corona-warn-app/cwa-event-qr-code'>command line tool</a> to generate several QR codes at once, for example.",
+        "introCLI": " You can create QR codes in the app or here on this page. Alternatively, you can also use this <a href='https://github.com/corona-warn-app/cwa-event-qr-code'>command line tool</a> to generate several QR codes at once, for example.",
```
Comment (heinezen) [inline src/data/eventregistration.json:7]: Can you change "this command line tool" to "our official QR code utilities"? That should be more precise.

Comment (Ein-Tim) [reply src/data/eventregistration.json:7]: Yep, I can do this. I'd also change the German wording then, ok?

Comment (heinezen) [reply src/data/eventregistration.json:7]: Yes, please :)

Comment (Ein-Tim) [reply src/data/eventregistration.json]: Done in [`46472d8` (#1591)](https://github.com/corona-warn-app/cwa-website/pull/1591/commits/46472d81115d8780639bd05e69fae4f94ea7b103).

**Comment by MikeMcC399** [comment]
@Ein-Tim 
On https://github.com/corona-warn-app/cwa-event-qr-code/blob/main/README.md I can only see the one utility `cwa-event-qr-code`.

Maybe I missed something, but if there is only one utility, then you should be referring to it in the singular (utility/Werkzeug), not plural (utilities/Werkzeuge).

I'm not sure why @heinezen suggested to use "utilities".

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I agree with you, but before I change the wording, I'd like to hear feedback from @heinezen.

**Comment by heinezen** [comment]
@Ein-Tim @MikeMcC399 The sidebar uses singular "utility", so Mike is correct :)

**Comment by heinezen** [comment]
@Ein-Tim @MikeMcC399 

I've made the changes myself.

Btw, I've reduced the hours I work on this project, so my shifts will be very irregular. Therefore, if you ask me for feedback, it could take a very long time :D

----
Corona-Warn-App Open Source Team

**Pull request merged**